client/src/app/my-projects/my-projects.component.ts

```diff
@@ -55,12 +54,8 @@ export class MyProjectsComponent implements OnInit, OnDestroy, AfterViewInit {
   ngOnInit() {
     this.isLoggedIn$ = this.aai.isUserLoggedIn();
     this.account$ = this.isLoggedIn$.pipe(
-      concatMap(loggedIn => {
-        if (loggedIn) {
-          return this.ingestService.getUserAccount();
-        }
-        return of(undefined);
-      })
+      filter(loggedIn => loggedIn.valueOf()),
```

Comment:
For brevity, could just do `filter(loggedIn => loggedIn)` or `filter(identity)` (would need to import `identity` from rxjs). 

But for readability sake, I'd just do `filter(loggedIn => loggedIn === true)`

Reply:
> `filter(loggedIn => loggedIn)`

Wasn't allowed because of boolean != Boolean

I'll replace with `filter(loggedIn => loggedIn === true)`